lua/startup.lua

```diff
@@ -4,6 +4,15 @@ require 'menu'
 require 'math' 
 math.randomseed(os.time()) -- more random
 
+local fd = io.open(os.getenv("HOME").."/version.txt","r")
+if fd then
+  io.input(fd)
+  norns.version.update = io.read()
+  io.close(fd)
+else
+  norns.version.update = "000000"
```

Comment:
this can be initialized in norns.lua too

Reply:
will do